README.md

```diff
@@ -143,19 +143,25 @@ import Get from "restful-react";
 
 export default () => (
   {/* Use the lazy prop to not send a request */}
-  <Get path="/breeds" lazy>
+  <Get path="breeds" lazy>
     {data => {
       return (
         <div>
           <h1>Random Image</h1>
           {/* Composes path with parent: sends request to /breeds/image/random */}
-          <Get path="/image/random">
+          <Get path="image/random">
+            {image => <img alt="Random Image" src={image && image.message} />}
+          </Get>
+
+          <h1>Random Breed</h1>
+          {/* Escapes the relative path with parent: sends request to /random */}
+          <Get absolute path="random">
```

Comment:
This is the only documented place for the `absolute` prop, should we add a more detailed description to emphasize its importance?

Reply:
In the props interface. :wink: